document/src/main/java/com/ritense/document/domain/impl/searchfield/SearchField.java

```diff
@@ -16,8 +16,8 @@
 
 package com.ritense.document.domain.impl.searchfield;
 
-import com.fasterxml.jackson.annotation.JsonValue;
 import org.springframework.data.domain.Persistable;
+
```

Comment:
entity properties should have the same constraints according to the constraints added to the properties made in liquibase migration(document/src/main/resources/config/liquibase/changelog/20221021-add-search-field-table.xml)

Reply:
I've added them